packages/create-next-app/create-app.ts

```diff
@@ -164,6 +164,14 @@ export async function createApp({
       JSON.stringify(packageJson, null, 2) + os.EOL
     )
 
+    // Copy Yarn.lock file
+    if (useYarn) {
```

Comment:
Bear in mind you'll probably want something like https://github.com/facebook/create-react-app/pull/6253 to ensure this doesn't break private npm registry setups.